integration-tests/tests/ubuntuFan_test.go

```diff
@@ -1,5 +1,5 @@
 // -*- Mode: Go; indent-tabs-mode: t -*-
-// +build !excludeintegration
+// +build !excludeintegration,!excludeflaky
```

Comment:
Sounds good, I won't block this any further then. LGTM. Thanks for your time explaining the issue.